### PR TITLE
Sort mean values, Update images

### PR DESCRIPTION
**Pull request by andy-sgro**
- getSyncData() autofills the trait descriptions.
- Added Barplot.drawMeanPlot().
- Moved data averaging functions to the Data class.
- Replaced SDSN logo with png version.
- No longer multiplying left-side bar plot by 100.
- Added unit tests for Data.getMeanCity().
#2

### CODE REVIEW

File: src/barplot.ts
Comment (jamie-sgro) [inline src/barplot.ts:328]: Great idea to reduce that long string. The right graph will later take input as strings, such as a ratio, represented as "1:1"  or percent's, with the symbol, e.g. "75%". With that in mind the parseFloat would remove those important symbols. The dev branch has a new property for `DataPoint`, called `nonNormalizedValue` which is used for the tooltip. Let's hold off on including this parse from this branch, and instead have the averaging methods also add that rounded value to the DataPoint as a `nonNormalizedValue`

File: src/main.ts
Comment (jamie-sgro) [inline src/main.ts:71]: The [100, 100] you see here is a hardcoded max value, we could change it to [1,1] if you wanted to have the scale be between 0-1. This value should be inferred later on rather than hardcoded anyways

Comment (jamie-sgro) [reply src/main.ts:71]: Specifically is sets the min and max for the x-axis

File: src/main.ts
Comment (jamie-sgro) [inline src/main.ts:64]: From [this post ](https://github.com/jamie-sgro/dashboard/pull/12/files#r670824674), we don't need to * 100 here, we just need to change the range

File: src/main.ts
Comment (jamie-sgro) [inline src/main.ts:91]: Can confirm from my hardcoded averages this morning, we're getting the exact same values, well done! Could we draft up some unit tests for this method? Let me know if you'd like to pair program it as well.

**Review by jamie-sgro — CHANGES_REQUESTED** [review]
Fantastic work., wonderful implementation of the sorting high-order function as well. Only requesting change to add unit tests to the averaging method and to revert the `Barplot` code to parseInt since another branch will resolve that error

File: src/barplot.ts
Comment (andy-sgro) [reply src/barplot.ts:328]: Good suggestion, going to reverse the parseFloat() line.

File: src/main.ts
Comment (andy-sgro) [reply src/main.ts:71]: I'll see about using the max average value as a potential max x-value.

File: public/home.html
Comment (jamie-sgro) [inline public/home.html:171]: Great call using height as the common denominator to resolve that oddly shaped image. Take a look at the issue specs, we'll want to replace the .jpg with the .png from the email I sent you. You'll notice a difference as we zoom the image like below. I've emailed the png for reference

image

File: src/main.ts
Comment (andy-sgro) [reply src/main.ts:64]: Will remove the '* 100' and add unit tests to the averaging functions.

File: src/main.ts
Comment (andy-sgro) [reply src/main.ts:91]: Thanks! I will draft up some unit tests and run them by you for approval.

File: public/home.html
Comment (andy-sgro) [reply public/home.html:171]: Thanks for the email, I will integrate it soon.

Comment (jamie-sgro) [reply public/home.html]: resolved! awesome work

**Review by jamie-sgro — APPROVED** [review]
Awesome work moving the considerations out of main.ts and into Barplot and Data respectively. I also love the work you did further normalizing the data flat file by adding a switch statement on the description. For the switch I'm not including it in this PR since upstream changes this schema somewhat, but I'll be implementing the same imminently. Awesome work once again, you're approved! Go ahead and merge

**Pull request merged**
